mdaviz/mainwindow.py

```diff
@@ -229,11 +226,21 @@ def get_all_subfolders(
                 settings.setKey(DIR_SETTINGS_KEY, ",".join(recent_dirs))
 
             else:
-                self._folderPath = None
-                self._dataPath = None
-                self.setSubfolderList([])
                 comment = f"{str(folder_path)!r} - invalid path."
-                self.folderNotValid(layout, comment)
+                self.folderNotValid(comment)
+
+    def folderNotValid(self, comment):
```

Comment:
This name (`folderNotValid`) implies a test that returns a boolean.

Reply:
does `ifFolderNotValid` work better?

Reply:
Looks it is [called ](https://github.com/BCDA-APS/mdaviz/blob/75c536fb9960840d59a9523c8a898df6aef6b20d/mdaviz/mainwindow.py#L230)once a selected folder has been determined as not valid.

Reply:
I just got rid of that method since it is only called once.